Water-Physics/Assets/Code/Scripts/Environment/Waves.cs

```diff
@@ -0,0 +1,177 @@
+﻿using System;
+using UnityEngine;
+
+namespace Code.Scripts.Environment
+{
+    public class Waves : MonoBehaviour
+    {
+        public int dimension = 10;
+        public Octave[] octaves;
+        public int uvScale = 1;
```

Comment:
Gebruik `[SerializedField] private` om iets in de inspector te tonen als het niet public hoeft.

Reply:
Veranderd `dimension` de size van het water? Ik zag geen verschil nadat ik dit in de inspector had aangepast?